src/scripts.js

```diff
@@ -0,0 +1,18 @@
+var saveButton = document.querySelector(".save-button")
+
+saveButton.addEventListener('click', genColor)
+
+    function genColor(){
```

Comment:
- Can be removed
- function genColor() is now a method within the class Colors.